### ISSUE TITLE
Feature Request: Option to show completed todos

### ISSUE THREAD

**Issue by harmtemolder** [larslockefeer/obsidian-plugin-todo#23]
The option to show completed todos would be nice. Maybe a toggle next to the 4 tabs?

**Comment by janpeeters70** [comment]
Yes please, this would come in very handy when you've accidentally completed a to do (like I just did and had no way to find it back). Thanks!

**Comment by jgarr16** [comment]
Would be a great way to find those items that you've purged from memory... after the GTD fact! 👍🏻

**Comment by RedemptionC** [comment]
+1
